app/javascript/components/region-form/index.jsx

```diff
@@ -0,0 +1,67 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import { Grid } from 'patternfly-react';
+import createSchema from './region-form.schema';
+import MiqFormRenderer from '../../forms/data-driven-form';
+import { API } from '../../http_api';
+import miqRedirectBack from "../../helpers/miq-redirect-back";
+
+const RegionForm = ({ id, maxDescLen }) => {
+  const [description, setDescription] = useState({});
+
+  useEffect(() => {
+    miqSparkleOn();
+    API.get(`/api/regions/${id}?attributes=description`)
+      .then(({ description }) => setDescription({ description }))
+      .then(miqSparkleOff);
+  }, [id]);
+
+  const onCancel = () => {
+    const message = __('Edit of Region was cancelled by the user');
+    miqRedirectBack(message, 'success', `/ops/explorer/?button=cancel`);
+  };
+
+  const onSubmit = (values) => {
+    miqSparkleOn();
+    const message = __('Region was saved');
+    const submitUrl = `/ops/explorer/${id}?button=save`;
+
+    API.post(`/api/regions/${id}`, {
+      action: 'edit',
+      resource: {
+        description: values.description,
+      }
+    })
+      .then(() => miqRedirectBack(message, 'success', submitUrl))
+      .catch(miqSparkleOff);
+  };
+
+  if (!description) {
```

Comment:
If you have an empty object for `description`, this will be always false. That's why we need the variable value to just a flat string everywhere else except below in the render.